In [9]:
# Import Necessary Libraries
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns

In [10]:
#reading the data into python environment and assigning the index column as 'ID'.

#train dataset
train=pd.read_csv(r'C:\Users\DELNA.RAPHEL\Desktop\DSA\Analytics Vidhya\DATA\train_PDjVQMB.csv',index_col='ID')

#test dataset
test=pd.read_csv(r'C:\Users\DELNA.RAPHEL\Desktop\DSA\Analytics Vidhya\DATA\test_lTY72QC.csv',index_col='ID')

#sample submission dataset
sam=pd.read_csv(r'C:\Users\DELNA.RAPHEL\Desktop\DSA\Analytics Vidhya\DATA\sample_OoSmYo5.csv',index_col="ID")

In [11]:
#displaying the train data
train.head()

Age  Gender         Income     Balance  Vintage  Transaction_Status  \
ID                                                                              
84e2fcc9   36  Female       5L - 10L   563266.44        4                   0   
57fea15e   53  Female   Less than 5L   875572.11        2                   1   
8df34ef3   35  Female  More than 15L   701607.06        2                   1   
c5c0788b   43  Female  More than 15L  1393922.16        0                   1   
951d69c4   39  Female  More than 15L   893146.23        1                   1   

         Product_Holdings  Credit_Card Credit_Category  Is_Churn  
ID                                                                
84e2fcc9                1            0         Average         1  
57fea15e                1            1            Poor         0  
8df34ef3                2            0            Poor         0  
c5c0788b                2            1            Poor         1  
951d69c4                1            1            Good         1

In [12]:
train.info()

<class 'pandas.core.frame.DataFrame'>
Index: 6650 entries, 84e2fcc9 to 57b39511
Data columns (total 10 columns):
 #   Column              Non-Null Count  Dtype  
---  ------              --------------  -----  
 0   Age                 6650 non-null   int64  
 1   Gender              6650 non-null   object 
 2   Income              6650 non-null   object 
 3   Balance             6650 non-null   float64
 4   Vintage             6650 non-null   int64  
 5   Transaction_Status  6650 non-null   int64  
 6   Product_Holdings    6650 non-null   object 
 7   Credit_Card         6650 non-null   int64  
 8   Credit_Category     6650 non-null   object 
 9   Is_Churn            6650 non-null   int64  
dtypes: float64(1), int64(5), object(4)
memory usage: 571.5+ KB


- <b> There are 6650 rows and 10 columns in train data.</b>